### PR TITLE
Float format

### LINKED ISSUE #263: csv_precision not working as expected
## Description

When specifying csv_precision, the generated file doesn't reflect the requested csv_precision. For example, with ensemble_grid_point_wetdays, if I request a csv_precision of 5, I obtain a csv file with 1 decimal.

## Environment

* Finch version used, if any: v0.10.0

## Additional Information

The reason of this behavior, is that the usage of Numpy's round keeps the data within the same format (float stays float, double stays double). Instead, the float_format parameter of dataframe.to_csv() could be used, as I propose here: https://github.com/bird-house/finch/tree/float_format

### PR DESCRIPTION
**Pull request by perronld**
## Overview

This PR fixes #263

Changes:

* Manually convert values to string using formatting instead of round() when csv_precision is requested

### CODE REVIEW

**Review by aulemahal — APPROVED** [review]
Looks good indeed!

**Review by aulemahal — APPROVED** [review]
I missed a breaking detail!

**Review by aulemahal — CHANGES_REQUESTED** [review]
(woups my second review meant to be "request changes", so here's a 3rd :sweat: )

File: finch/processes/ensemble_utils.py [hunk 1]
```diff
@@ -634,7 +634,14 @@ def ensemble_common_handler(
         if "region" in df.columns:
             df.drop(columns="region", inplace=True)
 
-        df.dropna().to_csv(ensemble_csv)
```
[the remark this answers is not in the document]
Comment (perronld) [reply finch/processes/ensemble_utils.py:637]: Yes, I got a request to keep those lines here: https://ccdpwiki.atlassian.net/browse/CLIM-287

**Review by aulemahal — APPROVED** [review]

**Pull request merged**